core-java-8/src/test/java/com/baeldung/datetime/AgeCalculatorUnitTest.java

```diff
@@ -0,0 +1,49 @@
+package com.baeldung.datetime;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.LocalDate;
+import java.util.Date;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class AgeCalculatorUnitTest {
+
+    private final int EXPECTED_AGE = 72;
+
+    private final int DOB_YEAR = 1947;
+    private final int DOB_MONTH = 8;
+    private final int DOB_DAY = 15;
+
+    private final int YEAR = 2020;
+    private final int MONTH = 7;
+    private final int DAY = 12;
+    
+    private final String DATE_FORMAT = "yyyyMMdd";
+

```

Comment:
Might be good to add a unit test to show what happens when currentDate precedes birthDate, or add a validation check in your methods to prevent it.

Reply:
Done :)

Reply:
Done :-)